tests/tests.sh

```diff
@@ -18,6 +18,13 @@ sssom split $INPUT_URL_1 --output-directory $OUTPUT_DIR
 sssom convert $INPUT_FILE_1 --output $OUTPUT_DIR/converted_basic.tsv --output-format tsv
 sssom convert $INPUT_URL_1 --output $OUTPUT_DIR/converted_basic.tsv --output-format tsv
 
+for fmt in tsv json owl rdf
+do
+  sssom convert $INPUT_FILE_1 --output $OUTPUT_DIR/converted_basic.tsv --output-format $fmt
```

Comment:
This will output all files with extension '.tsv'. 
Proposed change:
```
sssom convert $INPUT_FILE_1 --output $OUTPUT_DIR/converted_basic.$fmt --output-format $Mt
```

I tried this but the .rdf file shows an error in Mozilla.

Reply:
Committed this change.